### ISSUE TITLE
Add properties other than the exampled properties

### ISSUE THREAD

**Issue by eran-pinhas** [internetitem/logback-elasticsearch-appender#37]
If you want to add another properties such as "isSuccess", "calulationTime" etc.

**Comment by PeterFokkinga** [comment]
@eran-pinhas could you elaborate? Ie. is this a feature request or do you need more documentation / examples?

The property mechanism is already quite flexible, for example I have:
```
<property>
        <name>host</name>
        <value>${applicationServerName}</value>
        <allowEmpty>false</allowEmpty>
</property>
 <property>
        <name>httpParameters</name>
        <value>%mdc{httpParameters}</value>
        <type>object</type>
        <allowEmpty>false</allowEmpty>
</property>
```
with `  <define name="applicationServerName" class="nl.rug.lib.blackboard.logging.AppserverDefiner" />` where `AppserverDefiner` implements the `ch.qos.logback.core.PropertyDefinerBase` interface, and `httpParameters` is set by using `MDC.put("httpParameters", ...)` in my servlet code

The "object" type is something I'm working on right now, it is not available in the current (1.6) release; I will have a pull request shortly.

**Comment by eran-pinhas** [comment]
Oh, it was long time ago. It wat feature request, and you have provided it so I'm closing the issue
